app/components/HomeHeader/styles.js

```diff
@@ -0,0 +1,45 @@
+import {StyleSheet} from 'react-native';
+
+const styles = StyleSheet.create({
+  header: {
+    height: 70,
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    paddingTop: 10,
+    paddingRight: 5,
+    width:'100%', 
+    backgroundColor:'#E07E5B',
```

Comment:
maintain consistency about adding comma at end of the styles. If you add ','  or not at end be sure to do same in all other styles.